sdk/tables/Azure.Data.Tables/tests/samples/Sample8_CustomizingSerialization.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Data.Tables.Tests;
+using System.Net;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+using Azure.Identity;
+
+namespace Azure.Data.Tables.Samples
+{
+    [LiveOnly]
+    public partial class TablesSamples : TablesTestEnvironment
```

Comment:
If you inherit from the `SamplesBase<>` you wouldn't need to ifdef the `DefaultAzureCredential`

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.Query/tests/MetricsQueryClientSamples.cs#L28